editor/src/clj/editor/app_view.clj

```diff
@@ -1318,12 +1318,11 @@ If you do not specifically require different script states, consider changing th
           render-save-progress! (make-render-task-progress :save-all)
           render-build-progress! (make-render-task-progress :build)
           task-cancelled? (make-task-cancelled-query :build)
-          build-server-headers (native-extensions/get-build-server-headers prefs)
-          bob-args (bob/build-html5-bob-args project prefs)
+          bob-args (bob/build-html5-bob-options project prefs)
```

Comment:
build server headers are now assigned to Bob options directly, inside the `build-html5-bob-options` fn.